docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -118,7 +118,7 @@ A draft implementation is optional, but you may find it useful to create one bef
 
 ### Resources
 
-- Start with the [Spec Template](Spec-Template)
+- Start with the [Spec Template](https://github.com/microsoft/fluentui/blob/master/docs/react-wiki-archive/Spec-Template.md)
```

Comment:
we should hoist that to normal react-v9 docs rather having a spec template in archived wiki